src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -50,6 +51,21 @@ public static Name parseName(String name) throws ParseException {
         return new Name(trimmedName);
     }
 
+    /**
+     * Parses a {@code String name} into a {@code FolderName}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code folderName} is invalid.
+     */
+    public static FolderName parseFolderName(String folderName) throws ParseException {
```

Comment:
should we add some test cases to `ParserUtilTest.java`?

Reply:
I agree. I've added test cases for the parseFolderName method!